docs/configuration.asciidoc

```diff
@@ -508,7 +504,7 @@ Use this option to ignore certain URL patterns eg. healthchecks or admin section
 | `ELASTIC_APM_INSTRUMENTED_RAKE_TASKS` | `instrumented_rake_tasks` | `[]`    | `['my_task']`
 |============
 
-Elastic APM can instrument your Rake tasks but given that they are used for a multitude of sometimes very different and not always relevant things, this is opt in.
+Elastic APM can instrument your Rake tasks. Theis is an opt-in field, as they are used are for a multitude of things. 
```

Comment:
Does this make sense or is it changing the meaning too much?

Reply:
```suggestion
Elastic APM can instrument your Rake tasks. Because Rake tasks are used for very different things, this is an opt-in field.
```